lib/tasks/evm_application.rb

```diff
@@ -19,15 +19,12 @@ def self.start
     _log.info("EVM Startup initiated")
 
     MiqServer.kill_all_workers
-    rr      = File.expand_path(Rails.root)
-    runner  = File.join(rr, "bin/rails runner")
-    program = File.join(rr, "lib/workers/bin/evm_server.rb")
-    command_line = "#{runner} #{program}"
+    command_line = "#{Gem.ruby} #{Rails.root.join("lib/workers/bin/evm_server.rb").expand_path}"
 
     env_options = {}
     env_options["EVMSERVER"] = "true" if MiqEnvironment::Command.is_appliance?
     puts "Running EVM in background..."
-    pid = Kernel.spawn(env_options, command_line, :pgroup => true, [:out, :err] => [Rails.root.join("log/evm.log"), "a"])
+    pid = Kernel.spawn(env_options, command_line.to_s, :pgroup => true, [:out, :err] => [Rails.root.join("log/evm.log"), "a"])
```

Comment:
Good call, originally it was a Pathname and spawn's c code doesn't accept pathnames.